Pkgfile

```diff
@@ -3,18 +3,19 @@
 format: v1alpha2
 
 vars:
-  # renovate: datasource=github-tags extractVersion=^v(?<version>.*)$ depName=ARM-software/arm-trusted-firmware
```

Comment:
let's add two version of ARM ATF instead of removing this one and reference the new one for r5s only

Reply:
Done